traefik/templates/rbac/clusterrolebinding.yaml

```diff
@@ -1,4 +1,9 @@
+{{- if .Values.rbac.enabled -}}
+{{- if .Values.rbac.namespaced }}
```

Comment:
Could you please split that into 2 files:
- clusterrolebindin.yaml
- rolebindin.yaml

Reply:
Done